home/tests/test_login.py

```diff
@@ -41,3 +41,8 @@ def test_authenticated_user_view(self, client, valid_user_details):
         response = client.get('/')
         assert "Login" not in str(response.content)
         assert "Logout" in str(response.content)
+
+    @pytest.mark.parametrize(('next'), [('about'), ('explore'), ('tags')])
+    def test_redirection_to_last_page_url(self, client, next, valid_user_details):
+        response = client.post('/login/?next=/{0}/'.format(next), valid_user_details)
+        assert response.url == '/{0}/'.format(next)
```

Comment:
I think you should add `assert response.status_code == 302`

Reply:
I have this assertion in `test_success_login_post_redirect`